docs/getstarted/themes.md

```diff
@@ -26,7 +26,7 @@ The active color theme is stored in your user [settings](/docs/getstarted/settin
 
 ```json
   // Specifies the color theme used in the workbench.
-  "workbench.colorTheme": "Default Dark+"
+  "workbench.colorTheme": "Dark Modern"
```

Comment:
The setting value is "Default Dark Modern"

Reply:
This section is calling out the underlying settings as a JSON snippet so it should reflect the actual setting JSON value.